models/banana_gun/solana/banana_gun_solana_trades.sql

```diff
@@ -0,0 +1,167 @@
+{{ config(
+    alias = 'trades',
+    schema = 'banana_gun_solana',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['blockchain', 'tx_id', 'tx_index', 'outer_instruction_index', 'inner_instruction_index']
+   )
+}}
+
+{% set project_start_date = '2024-01-08' %}
+{% set fee_receiver_1 = '8r2hZoDfk5hDWJ1sDujAi2Qr45ZyZw5EQxAXiMZWLKh2' %}
+{% set fee_receiver_2 = 'Cj297UauzMX64FU9dKJZRUBWszJ7tEWpVheasq4CfATV' %}
+{% set fee_receiver_3 = 'HKMh8nV3ysSofRi23LsfVGLGQKB415QAEfZT96kCcVj4' %}
+{% set fee_receiver_4 = '7tQiiBdKoScWQkB1RmVuML7DBGnR31cuKPEtMM7Vy5SA' %}
+{% set fee_receiver_5 = '4BBNEVRgrxVKv9f7pMNE788XM1tt379X9vNjpDH2KCL7' %}
+{% set wsol_token = 'So11111111111111111111111111111111111111112' %}
```

Comment:
Maybe we can design a macro for solana bot trades.

Reply:
what would this look like? also not all bots handle fees in the same way, so the logic to find their trades can differ from bot to bot

Reply:
Let me study these cases first...

Reply:
let's not force a macro, if it requires too many "if this, then that" type coding within. take an hour or two to dig into this @Hosuke, see if possible, then decide if it's worth the effort or not

it's more important that we follow the design for efficiency in orchestration at the end of the day (base level raw data, metadata joins later downstream, etc etc)